src/components/Person/Person.jsx

```diff
@@ -1 +1,20 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => {
+  let messege = '';
+
+  if (!person.isMarried) {
+    messege = `I am not married`;
+  } else {
+    messege = `${person.partnerName} is my ${person.sex === 'm' ? 'wife' : 'husband'}`;
+  }
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">{`My name is ${person.name}`}</h2>
+      {person.age === undefined
+        ? ''
+        : <p className="Person__age">{`I am ${person.age}`}</p>
+      }
+      <p className="Person__partner">{messege}</p>
```

Comment:
```suggestion
      <p className="Person__partner">{!isMarried ? 'I am not married' : `${partnerName} is my ${partnerStatus}`}</p>
```

Reply:
This is less readable, so another way was chosen

Reply:
it's pretty readable
construction you used is a little bit bulky 
```
  let messege = '';

  if (!person.isMarried) {
    messege = `I am not married`;
  } else {
    messege = `${person.partnerName} is my ${person.sex === 'm' ? 'wife' : 'husband'}`;
  }
```
so now it looks nice